semconv-incubating/src/main/java/io/opentelemetry/semconv/incubating/MessagingIncubatingAttributes.java

```diff
@@ -35,7 +35,7 @@ public final class MessagingIncubatingAttributes {
       longKey("messaging.batch.message_count");
 
   /** A unique identifier for the client that consumes or produces a message. */
-  public static final AttributeKey<String> MESSAGING_CLIENT_ID = stringKey("messaging.client_id");
+  public static final AttributeKey<String> MESSAGING_CLIENT_ID = stringKey("messaging.client.id");
```

Comment:
that's the demo of the resolution:
- `messaging.client_id` is deprecated, `messaging.client.id` is not 
- `messaging.client_id` is completely ignored during codegen

This results in potential mismatch between schema url and attribute values, but we would not rename a stable attribute, but it seems tolerable for experimental.

Reply:
no relevant anymore